ui/mirage/fixtures/allocations.js

```diff
@@ -0,0 +1,19 @@
+export default [
+  {
+    name: 'fixture-array-g-0.[0]',
+    taskGroup: 'fixture-array-g-0',
+    jobId: 'job-100',
+    useMessagePassthru: false,
+    desiredStatus: 'evict',
+    clientStatus: 'running',
+    modifyTime: 1516096891447000000,
+    jobVersion: 1,
+    modifyIndex: 1879,
+    id: 'fixture-fe95bbed-ceb3-4d17-8eee-aae324d130c2',
+    nodeId: '43d9dc36-7cd4-4abc-982b-e978ec3e6fcd',
+    taskStateIds: ['1000'],
+    task_state_ids: ['1000'],
+    taskResourcesIds: ['1'],
+    taskResourceIds: ['1'],
+  },
+];
```

Comment:
Although it's incredibly helpful that Mirage let's you use static fixtures like this, I'm not ready to commit any to source control. I think more thought is necessary to determine how to do this in a generally applicable way.